README.md

```diff
@@ -17,14 +17,39 @@ The latest image is available on DockerHub at `listmonk/listmonk:latest`. Use th
 
 ```bash
 mkdir listmonk-demo
-sh -c "$(curl -sSL https://raw.githubusercontent.com/knadh/listmonk/master/install-demo.sh)"
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/knadh/listmonk/master/install-demo.sh)"
 ```
 
 The demo does not persist Postgres after the containers are removed. DO NOT use this demo setup in production.
 
 #### Production
+
+##### Easy Install
+
+This setup is recommended if you want to _quickly_ setup `listmonk` in production.
+
+```bash
+mkdir listmonk
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/knadh/listmonk/master/install-prod.sh)"
+```
+
+The above shell script performs the following actions:
+
+- Downloads a sample `config.toml` and `docker-compose.yml`.
```

Comment:
I think slightly better language would be:

- Downloads `docker-compose.yml` and generates a `config.toml`.
- Runs a Postgres container and installs the database schema.
- Runs the listmonk container.